controllers/v1/cloud-services/gcp.js

```diff
@@ -180,5 +180,96 @@ module.exports = class Gcp {
     })
    }
 
+
+       /**
+     * @api {post} /kendra/api/v1/cloud-services/gcp/uploadFile  
+     * upload file to presignedUrl
```

Comment:
@rakeshSgr - Make upload as upload file to presignedUrl, => File upload to GCP

Reply:
resolved